internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixCustomizedViewOfflinePushRepository.java

```diff
@@ -128,22 +129,9 @@ protected void onCustomizedViewDataChange(RoutingTableSnapshot routingTableSnaps
       Set<String> instancesSeenInCustomizedViewButMissingFromLiveInstances = new HashSet<>();
       for (CustomizedView customizedView: customizedViewCollection) {
         String resourceName = customizedView.getResourceName();
-        int partitionCount = resourceToPartitionCountMap.getOrDefault(resourceName, -1);
-        if (partitionCount == -1) {
-          String storeName = Version.parseStoreFromKafkaTopicName(resourceName);
-          int version = Version.parseVersionFromVersionTopicName(resourceName);
-          Store store = storeRepository.getStore(storeName);
-          if (store == null) {
-            LOGGER.warn("Cannot find store for resource: {}.", resourceName);
-            continue;
-          }
-
-          if (!store.getVersion(version).isPresent()) {
-            LOGGER.warn("Version not found in store for resource: {}.", resourceName);
-            continue;
-          }
-          partitionCount = store.getVersion(version).get().getPartitionCount();
-          resourceToPartitionCountMap.put(resourceName, partitionCount);
+        int partitionCount = getPartitionCount(resourceName);
+        if (partitionCount == STORE_VERSION_PARTITION_COUNT_NOT_FOUND) {
+          continue;
```

Comment:
Are we no longer interested in logging the above cases?

Admittedly, I'm not sure where we'd be in a position where we'd have a customized view entry and no discernable store config/version.  My guess is it could spring up if we were mid version deletion?  Possible, but not very interesting I guess.

Reply:
Nevermind, I see you just moved them to the function.